website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -54,7 +54,7 @@ Viewing a workspace (individually or in a list) requires permission to read runs
 | -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | `:organization_name` | The name of the organization to create the workspace in. The organization must already exist in the system, and the user must have permissions to create new workspaces. |
 
--> **Note:** Workspace creation is restricted to the owners team, teams with the "Manage Workspaces" permission, and the [organization API token](/terraform/cloud-docs/users-teams-organizations/api-tokens#organization-api-tokens).
+-> **Note:** Workspace creation is restricted to the owners team, teams with the "Manage all workspaces" permission, and the [organization API token](/terraform/cloud-docs/users-teams-organizations/api-tokens#organization-api-tokens).
```

Comment:
Actually, should "Manage all projects" be included in this note, too? If that [permission level has all the of permissions of "Manage all workspaces"](https://terraform-docs-common-git-juliannatetreault-up-43c436-hashicorp.vercel.app/terraform/cloud-docs/users-teams-organizations/permissions#manage-all-projects)?

Reply:
I've updated this note -- let me know what you think, @rkoron007!

Reply:
Looks perfect; thank you so much!